src/main/java/version1/gameUtil/screens/LeaderBoardScreen.java

```diff
@@ -35,4 +231,38 @@ public void buildBody() {
     public void ready() {
 
     }
+
+    /**
+     * Creates the go to menu game mode button
+     * @return this
+     */
+    public LeaderBoardScreen createGoToMenuButton(String label){
+        /*
+         * TODO: Add a gotToModeScreen logic : see List of buttons at the top  | Nicole
```

Comment:
This comment should say add goToMenuScreen logic, and not goToModeScreen. My bad